.github/scripts/upload_tests_results.py

```diff
@@ -0,0 +1,342 @@
+            end = min(start + batch_size, total_records)
+            batch = results[start:end]
+
+            sql = f"""--!syntax_v1
+            UPSERT INTO `{path}` 
+            (build_type,
+            job_name,
+            job_id,
+            commit,
+            branch,
+            pull, 
+            run_time, 
+            test_id, 
+            suite_name, 
+            raw_test_name, 
+            file, 
+            test_name, 
+            fixture, 
+            time, 
+            status,
+            status_description,
+            log,
+            logsdir,
+            stderr,
+            stdout,
+            owners) VALUES"""
+            values = []
+            for index, result in enumerate(batch):
+
+                values.append(
+                    f"""
+                ("{result['build_type']}", 
+                "{result['job_name']}", 
+                {result['job_id']}, 
+                "{result['commit']}",
+                "{result['branch']}",
+                "{result['pull']}", 
+                DateTime::FromSeconds({result['run_time']}),
+                "{result['pull']}_{result['run_time']}_{start+index}",
+                "{result['suite_name']}",
+                "{result['raw_test_name']}",
+                "{result['file']}",
+                "{result['test_name']}",
+                "{result['fixture']}",
+                {result['time']},
+                "{result['status']}",
+                "{result['status_description']}",
+                "{result['log']}",
+                "{result['logsdir']}",
+                "{result['stderr']}",
+                "{result['stdout']}",
+                "{result['owners']}")
+                """
+                )
+            sql += ", ".join(values) + ";"
+
+            futures.append(executor.submit(upload_results, pool, sql))
+
+        for future in as_completed(futures):
+            future.result()  # Raise exception if occurred
+
+
+def create_pool(endpoint, database):
+    driver_config = ydb.DriverConfig(
+        endpoint,
+        database,
+        credentials=ydb.credentials_from_env_variables()
+    )
+
+    driver = ydb.Driver(driver_config)
+    driver.wait(fail_fast=True, timeout=5)
+    return ydb.SessionPool(driver)
+
+
+def main():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--result-file', action='store', dest="result_file", required=True, help='XML with results of tests')
+    parser.add_argument('--build-type', action='store', dest="build_type", required=True, help='build type')
+    parser.add_argument('--commit', default='store', dest="commit", required=True, help='commit sha')
+    parser.add_argument('--branch', default='store', dest="branch", required=True, help='branch name ')
+    parser.add_argument('--pull', action='store', dest="pull",required=True, help='pull number')
+    parser.add_argument('--run-time', action='store', dest="run_time",required=True, help='time of test run start')
```

Comment:
Why do we need this argument?

Reply:
for handy statistic i have to know date time of test run